src/wp-includes/class-wp-dependencies.php

```diff
@@ -126,6 +126,18 @@ public function do_items( $handles = false, $group = false ) {
 		$handles = false === $handles ? $this->queue : (array) $handles;
 		$this->all_deps( $handles );
 
+		return $this->process_to_do_items( $group );
+	}
+
+	/**
+	 * Processes the items marked for to do, and their dependencies.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @param int|false $group Optional. Group level: level (int), no group (false).
```

Comment:
If optional, there needs to be a default value added for the` $group` param.

Reply:
Good catch. Copypasta. Removed in 643055be43.